model_compression_toolkit/qunatizers_infrastructure/common/node_quantization_dispatcher.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from typing import Dict, List
+
+from model_compression_toolkit.qunatizers_infrastructure.common.base_quantizer import BaseQuantizer
+
+
+class NodeQuantizationDispatcher:
+    def __init__(self,
+                 weight_quantizers: Dict[str, BaseQuantizer] = None,
```

Comment:
I think that everywhere else in the code we use "weights" not "weight", so we might want to be consistent to prevent unnecessary future syntax errors.